v2/workloadapi/option.go

```diff
@@ -0,0 +1,26 @@
+package workloadapi
+
+import (
+	"github.com/spiffe/go-spiffe/v2/logger"
+	"google.golang.org/grpc"
+)
+
+type Option interface{}
```

Comment:
any reason for using an `interface{}` here instead of a strong type like this?
```
type Option interface {
    applyOption(*cfg)
}
```

Reply:
I fully intend implementors to add unexported methods to these option interfaces as necessary. I didn't want to get proscriptive on the name or parameters prematurely. To your point, I should probably add a TODO comment inside each of them pointing that out.